### DISCUSSION TITLE
Using top infill ironing with post-process ironing

### DISCUSSION THREAD

**Discussion by Hagman89** [supermerill/SuperSlicer#1839] [General]
Im trying to acheive a perfectly smooth finish and figure my best bet is to have the smoothest top layer before ironing, hence why Im experimenting with "top infill ironing". Im getting close but still getting a slightly grainy texture, would it make sense that the post process ironing flow should be extremely low ie 2% considering the previous layer should already be technically flat?

**Comment by supermerill** [comment]
It depends. Do both only if monotonic + ironing post-process isn't good enough.
It's a bit hard to have a flat top surface.
Also, a flow too low may jam your hotend, as the plastic may conduct the heat up into the cold zone.
